backend/app/GraphQL/Mutations/CreateQuestion.php

```diff
@@ -15,6 +16,12 @@ public function __invoke($_, array $args)
     {
         $questionData = Arr::except($args, ['tags']);
 
+        $hasNoTeam = !isset($args['team_id']) || $args['team_id'] === null;
+
+        if ($hasNoTeam && (isset($args['is_public']) && $args['is_public'] === false)) {
```

Comment:
I think this can be refactored in a much cleaner way. Please confirm functionality and apply to all related
```suggestion
        if (!$args['team_id'] && !args['is_public']) {
```

Reply:
I have tried this code sir but it will not catch all the validation and will prompt an error if either of team_id and is_public are not filled since they're not required in the form.

![image](https://user-images.githubusercontent.com/114911074/223004226-ae692fd6-aae3-4099-a6ca-7fe57aaf8f33.png)

Reply:
Hmm okay my concern lang is it looks redundant already. Can you try

```suggestion
        if ($hasNoTeam && !args['is_public']) {
```

Reply:
I have tried adjusting the logic according to your suggestion, also I've retained the isset() since it will return an error if the variable is not defined
```suggestion
        if ($hasNoTeam && (isset($args['is_public']) && !$args['is_public'])) {
```

Reply:
@jescabillas Im fine with this. Mu work ra?

Reply:
yes sir @ziontavera it works fine on my end